src/cpp/server/server_context.cc

```diff
@@ -182,7 +185,14 @@ void ServerContext::TryCancel() const {
 }
 
 bool ServerContext::IsCancelled() const {
```

Comment:
In that case, I think that we were saved by the fact that TryPluck would just do nothing. Not sure, though, since mixing pluck and next could lead to undefined behavior.